src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sources/streamfactories/TwitterStreamFactory.scala

```diff
@@ -62,30 +64,22 @@ class TwitterStreamFactory(configurationManager: ConfigurationManager) extends S
       query = Some(query)
     )
 
-    def isOriginalTweet(status: Status) : Boolean = {
-      !status.isRetweet && status.getRetweetedStatus == null
-    }
+    val trustedSourceScreenNames = params.getTrustedSources.toSet
+    stream.filter(status=>{
+      def isOriginalTweet(status: Status) : Boolean = {
+        !status.isRetweet && status.getRetweetedStatus == null
+      }
 
-    params.getOrElse("trustedSourceFilterEnabled", "true").toString.toBoolean match {
```

Comment:
This whole change is basically getting rid of the `trustedSourceFilterEnabled` check and using `trustedSourceScreenNames.isEmpty` in its place.